lib/bitvec.cpp

```diff
@@ -60,6 +61,7 @@ bitvec &bitvec::operator>>=(size_t count) {
 }
 
 bitvec &bitvec::operator<<=(size_t count) {
+    assert(static_cast<int>(count) >= 0);
```

Comment:
A too-large size should result in a bad_alloc here (from expand), so if we want to catch bad_alloc and deal with it, this might cause problems.